tests/framework/installer/ceph_manifests_v1.4.go

```diff
@@ -1720,16 +1916,38 @@ spec:
     command: ["/tini"]
     args: ["-g", "--", "/usr/local/bin/toolbox.sh"]
     env:
-      - name: ROOK_ADMIN_SECRET
+      - name: ROOK_CEPH_USERNAME
         valueFrom:
           secretKeyRef:
             name: rook-ceph-mon
-            key: admin-secret
+            key: ceph-username
+      - name: ROOK_CEPH_SECRET
+        valueFrom:
+          secretKeyRef:
+            name: rook-ceph-mon
+            key: ceph-secret
+    securityContext:
+      privileged: true
     volumeMounts:
+      - mountPath: /dev
```

Comment:
I didn't think the toolbox had these mounts anymore. They aren't needed by the tests, right? Why were they added back in?

Reply:
As discussed in huddle, leaving these in for now because they *might* be needed. We can remove later if not.